modules/ROOT/pages/dataweave-extension-plugin.adoc

```diff
@@ -0,0 +1,341 @@
+= Installing and Using the DataWeave 2.0 Extension
+
+Install and use the DataWeave 2.0 extension in Visual Studio Code to develop, preview, test, and deploy custom mappings and modules into libraries in a DataWeave project. The extension also enables you to perform transformations similar to those you perform in the https://developer.mulesoft.com/learn/dataweave/[DataWeave Playground].
+
+== Before You Begin
+
+Before installing the DataWeave 2.0 extension, you must:
+
+* Install JDK (Java 8 minimum)
+* Install Apache Maven
```

Comment:
Any particular version? Perhaps the latest or the one recommended by VS Code?
Asking bc I had Maven installed and then was asked by Visual Studio Code if I wanted
to install the recommended version of it, which I did. The version was 17.0.2

Reply:
@martincousido

Reply:
We should require at least 3.6.3 which is the embedded version we use for internally to resolve some dependencies.